styles/index.css

```diff
@@ -0,0 +1,108 @@
+.red{
+    color: red;
+}
+
+.orange{
+    color: orange;
+}
+
+.yellow{
+    color:yellow;
+}
+
+.green{
+    color:green;
+}
+
+.teal{
+    color:teal;
+}
+
+
+  
+header {
+    text-align: center;
+    font-size: large;
+    font-weight: bold;
+    padding: 15px;
+}
+
+h4 {
+    font-size: 19px;
+}
+
+h3 {
+    font-size: 30px;
+}
+
+button {
+    font-size: 16px;  
+}
+
+#temp-display{
+    font-size: 50px;
+    font-weight: bold;
+    text-align: center;
+}
+
+select {
+    font-size: 18px;
+}
+
+#sky-garden {
+    padding-top: 30px;
+    padding-bottom: 30px;
```

Comment:
you can write padding on one line like `padding: 30px 0px 30px 0px`
the order is `padding: top left bottom right`